doc/glossary.rst

```diff
@@ -835,7 +835,14 @@ Class APIs and Estimator Types
         * :term:`predict` if :term:`inductive`
 
     density estimator
-        TODO
+         An :term:`unsupervised` estimation of input probability density
+         function. Most commonly used techniques are:
+
+        * :ref:`histogram`- visually represents data as defined bins
```

Comment:
When the glossary says "density **estimator**", should we mean a "scikit-learn estimator" or "anything that estimates a density"?

(I am trying to see if "histogram" should be listed here.)

Reply:
I think we should err towards stricter language, especially since scikit-learn doesn't really provide histogram tools.

Reply:
Sure happy to remove if there is agreement?
I had a look at the glossary intro to see if it would help:
> This glossary hopes to definitively represent the tacit and explicit conventions applied in Scikit-learn and its API, while providing a reference for users and contributors. It aims to describe the concepts and either detail their corresponding API or link to other relevant parts of the documentation which do so

I guess it would lean towards removing?

Reply:
> I guess it would lean towards removing?

From the glossary intro, I'll side with not including the reference to histograms.

Reply:
Done!